src/components/Asset.vue

```diff
@@ -111,7 +111,7 @@ export default {
         posterVariants: this.videoPoster ? this.videoPoster.variants : [],
         deviceFrame: this.deviceFrame,
         alt: this.asset.alt,
-        id: this.identifier,
+        id: this.identifier.replace('.', ''),
```

Comment:
I don't think this is strictly necessary.

Reply:
It's not, but it was giving me issues [using Jest](https://github.com/vuejs/vue-test-utils/issues/667), so I decided to add it just in case!

Reply:
You likely ran into a CSS selector in a Jest test thinking that the dot is a class name. You can fix this by updating the selector to escape the dot using a backslash so that it treats it as a normal dot character and not the start of a classname.

https://stackoverflow.com/questions/12310090/css-selector-with-period-in-id

Reply:
Escaping it in the test would be necessary and not in the template (just to clarify)

Reply:
Makes sense, thank you! https://github.com/apple/swift-docc-render/pull/807/commits/9cfb09966d7320a961bb4e3c78dd36aace0553b0